### ISSUE TITLE
[css-conditional] `@import url("test.css") supports()` validity

### ISSUE THREAD

**Issue by CanadaHonk** [w3c/csswg-drafts#8598]
There's no clarification in spec or issues that I could find that `@import url("test.css") supports()` is valid or invalid? If valid/parse-able, what behavior should it have?

It looks like `@supports()` is valid, but as per spec it isn't with `@import ... supports()` afaik? Might be an oversight. Thanks!

Spec docs:
- https://www.w3.org/TR/css-cascade-4/#conditional-import
- https://www.w3.org/TR/css-conditional-3/#typedef-supports-condition

**Comment by emilio** [comment]
Yeah, `@supports ()` matches `<general-enclosed>`, which is `( <any-value>? )`, but `supports()` doesn't match that because of how the grammar is written... Probably an oversight indeed? cc @tabatkins @fantasai
